docs/docs/user-guide/appendix/releasenotes-fw.md

```diff
@@ -1,5 +1,27 @@
 # Release Notes
 
+## BioNeMo Framework v2.2
+
+### New Features
+
+* GNN-Compose
```

Comment:
We are currently working on a more scope-appropriate naming for this effort. For the moment, could you change GNN-Compose to Small molecule featurization

Reply:
Amended and force-pushed the naming. (Still visible here, oh well! 🙃)